README.md

```diff
@@ -1 +1,44 @@
-# tf_datadog
\ No newline at end of file
+# terraform-datadog-monitor
+
+A Terraform module which contains a number of common configurations of monitors for Datadog.
```

Comment:
Rewrite:

A Terraform module repository which contains a number of common configurations ("submodules") of Datadog monitors.